dnf-behave-tests/features/steps/common/repodata_representation.py

```diff
@@ -0,0 +1,167 @@
+# -*- coding: utf-8 -*-
+
+class MetadataItem:
```

Comment:
Please, use new style classes for python2: `class MetadataItem(object)`
Just in case we needed to run the tests on python2.